.editorconfig

```diff
@@ -15,5 +15,8 @@ indent_style = tab
 [*.json]
 indent_size = 4
 
+[*.yml]
+indent_size = 2
```

Comment:
I find an indent of 4 too wide for YAML, but that's of course highly subjective. Can revert if you prefer 4 spaces.